to_srt.py

```diff
@@ -52,7 +52,7 @@ def append_subs(start, end, prev_content):
         append_subs(prev_time["start"], prev_time["end"], prev_content)
         prev_time = {"start": start, "end": end}
         prev_content = [content]
-    append_subs(start, end, prev_content)
+        append_subs(start, end, prev_content)

```

Comment:
I was using this the file attached. But I think the file was malformed. I tried making some other changes to the script but dind't get the desired result.
[Shameless -uk 1x01.dfxp.txt](https://github.com/isaacbernat/netflix-to-srt/files/204350/Shameless.-uk.1x01.dfxp.txt)